### PR TITLE
Added dsc.gg presence.

### PR DESCRIPTION
**Pull request by DiscordAnaxes**
Here some images of the presence working:

### CODE REVIEW

**Review by N0chteil — CHANGES_REQUESTED** [review]
---

File: websites/D/Dsc.gg/presence.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    } else if (document.location.pathname === ("/developers/about")) {
+      presenceData.details = `Viewing 💻 developer page`;
+      presenceData.buttons = [
+        {
+          label: "View Page",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname === ("/dashboard")) {
+      presenceData.details = `Viewing 👤 dashboard`;
```
Comment (N0chteil) [inline websites/D/Dsc.gg/presence.ts:49]: Use **"** instead of **`**

File: websites/D/Dsc.gg/presence.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      ];
+    } else if (document.location.pathname === ("/developers/about")) {
+      presenceData.details = `Viewing 💻 developer page`;
+      presenceData.buttons = [
+        {
+          label: "View Page",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname === ("/dashboard")) {
```
Comment (N0chteil) [inline websites/D/Dsc.gg/presence.ts:48]:
```code
document.location.pathname === "/dashboard"
```
You don't need the brackets

File: websites/D/Dsc.gg/presence.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    } else if (document.location.pathname.includes("/premium")) {
+      presenceData.details = `Viewing 💎 premium page`;
+      presenceData.buttons = [
+        {
+          label: "View Page",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname === ("/developers/about")) {
+      presenceData.details = `Viewing 💻 developer page`;
```
Comment (N0chteil) [inline websites/D/Dsc.gg/presence.ts:39]: Use **"** instead of **`**

File: websites/D/Dsc.gg/presence.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      ];
+    } else if (document.location.pathname.includes("/premium")) {
+      presenceData.details = `Viewing 💎 premium page`;
+      presenceData.buttons = [
+        {
+          label: "View Page",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname === ("/developers/about")) {
```
Comment (N0chteil) [inline websites/D/Dsc.gg/presence.ts:38]:
```code
document.location.pathname === "/developers/about"
```
You don't need the brackets

File: websites/D/Dsc.gg/presence.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      const search = document.getElementById("searchBar")?.getAttribute("value");
+      presenceData.details = `🔍 Searching for: ${search || "Nothing"}`;
+      presenceData.buttons = [
+        {
+          label: "View Results",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname === ("/about")) {
+      presenceData.details = `Viewing ✨ about page`;
```
Comment (N0chteil) [inline websites/D/Dsc.gg/presence.ts:23]: Use **"** instead of **`**

File: websites/D/Dsc.gg/presence.ts [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      presenceData.buttons = [
+        {
+          label: "View Page",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname.includes("/premium")) {
+      presenceData.details = `Viewing 💎 premium page`;
```
Comment (N0chteil) [inline websites/D/Dsc.gg/presence.ts:31]: Use **"** instead of **`**

File: websites/D/Dsc.gg/presence.ts [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    } else if (document.location.pathname.includes("/search")) {
+      const search = document.getElementById("searchBar")?.getAttribute("value");
+      presenceData.details = `🔍 Searching for: ${search || "Nothing"}`;
+      presenceData.buttons = [
+        {
+          label: "View Results",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname === ("/about")) {
```
Comment (N0chteil) [inline websites/D/Dsc.gg/presence.ts:22]:
```code
document.location.pathname === "/about"
```
You don't need the brackets

**Review by N0chteil — CHANGES_REQUESTED** [review]
Some suggestions

File: websites/D/Dsc.gg/dist/metadata.json [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    "tags": [
+      "discord",
+      "discord-service",
+      "discord-invite",
+      "dscgg"
+    ],
+    "url": "dsc.gg",
+    "description": {
+      "en": "Free vanity links for you discord servers, bots and templates!"
+    }
```
Comment (Bas950) [inline websites/D/Dsc.gg/dist/metadata.json:28]:
```suggestion
    "description": {
      "en": "Free vanity links for you discord servers, bots and templates!",
      "nl": "Gratis vanity-links voor je Discord servers, bots en templates!"
    }
```

File: websites/D/Dsc.gg/presence.ts [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+    } else if (document.location.pathname === "/dashboard") {
+      presenceData.details = "Viewing 👤 dashboard";
+    } else if (document.location.pathname.includes("/dashboard/l/")) {
+      const link = document.location.pathname.split("/dashboard/l/");
+      presenceData.details = `Editing 🔗 ${link[1]} link`;
+      presenceData.state = `🏓 Tab: ${location.href.replace(`https://dsc.gg/dashboard/l/${link[1]}#tab=`, " ")}`;
+      presenceData.buttons = [
+        {
+          label: "View Link",
+          url: `https://dsc.gg/${link[1]}`
```
Comment (Bas950) [inline websites/D/Dsc.gg/presence.ts:57]:
```suggestion
          label: "Visit Link",
          url: `https://dsc.gg/${link[1]}`
```

**Comment by ririxidev** [comment]
Sorry but: 
image

**Comment by DiscordAnaxes** [comment]
> Sorry but:
> image

Like the discords.com presence, would you like me to talk with the owner, regarding when the domain was bought and how many visitors they receive?

**Comment by ririxidev** [comment]
I mean domain is 1 year old so i dont think they bought it but ye :/

**Comment by DiscordAnaxes** [comment]
> I mean domain is 1 year old so i dont think they bought it but ye :/

I can confirm that the domain is more than a year old and owned by the service:
[!image](https://cdn.anaxes.xyz/kotzjj5299a)

**Comment by ririxidev** [comment]
You didnt have to, I already told you that I know that this domain is 1 year old...

**Comment by DiscordAnaxes** [comment]
> You didnt have to, I already told you that I know that this domain is 1 year old...

However, you stated, "I don't think they bought it".

**Comment by ririxidev** [comment]
I meant that I didn't think that they bought it from someone else

**Comment by DiscordAnaxes** [comment]
> I meant that I didn't think that they bought it from someone else

Oh ok. My bad.

**Pull request closed without merging**